### PR TITLE
Update MagicApiaryInventory.java

### PR DESCRIPTION
**Pull request by ClassixX**
Changed sidedness to match forestry apiary and fixed canExtractItem to return true for the product slots

### CODE REVIEW

**Review by Alastors — CHANGES_REQUESTED** [review]
> Changed sidedness to match forestry apiary

can you show us a picture first of what you mean by this?

**Comment by ClassixX** [comment]
image
These lines caused the slots for Princess, Queen and Drone to only be accessible from two sides, the top and bottom I believe. In original Forestry, those slots can be inserted to from the sides as well.

**Review by Alastors — APPROVED** [review]

File: src/main/java/magicbees/bees/MagicApiaryInventory.java [hunk 1]
```diff
@@ -81,12 +81,11 @@ public void setInventorySlotContents(int i, ItemStack itemStack) {
...
 
     public int[] getAccessibleSlotsFromSide(int side) {
-        if (side == 0 || side == 1) {
-            return new int[] { SLOT_QUEEN, SLOT_DRONE };
-        }
-        int[] slots = new int[SLOT_PRODUCTS_COUNT];
+        int[] slots = new int[SLOT_PRODUCTS_COUNT + 2];
+        slots[0] = SLOT_QUEEN;
+        slots[1] = SLOT_DRONE;
         for (int i = 0, slot = SLOT_PRODUCTS_START; i < SLOT_PRODUCTS_COUNT; ++i, ++slot) {
```
Comment (AnrDaemon) [inline src/main/java/magicbees/bees/MagicApiaryInventory.java:87]: `for (int i = 2, …`

Don't run unnecessary operations inside cycle.

Comment (Alastors) [reply src/main/java/magicbees/bees/MagicApiaryInventory.java:87]: That's code that already existed

Comment (AnrDaemon) [reply src/main/java/magicbees/bees/MagicApiaryInventory.java:87]: See the change below this line.

He runs `i + 2` inside cycle, where he could just set `i = 2` initially and save yourself a few CPU cycles.

Comment (Alastors) [reply src/main/java/magicbees/bees/MagicApiaryInventory.java:87]: That's a lot of noise over an omelette tbh

**Pull request merged**